client/jetpack-cloud/sections/partner-portal/primary/licenses/index.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'calypso/components/main';
+import CardHeading from 'calypso/components/card-heading';
+import DocumentHead from 'calypso/components/data/document-head';
+import LicenseList from 'calypso/jetpack-cloud/sections/partner-portal/license-list';
+
+export default function Licenses(): ReactElement {
+	const translate = useTranslate();
+
+	return (
+		<Main wideLayout={ true } className="licenses">
+			<DocumentHead title={ translate( 'Licenses' ) } />
+
+			<CardHeading size={ 36 }>{ translate( 'Licenses' ) }</CardHeading>
+
+			<LicenseList />
+		</Main>
+	);
+}
```

Comment:
This isn't a blocker, but I think we could keep the `/licenses/` folder instead of moving it into `/primary/`. Even though `primary` is the same name as the main component of the layout, I think besides that it doesn't tell much about the content of the folder. I'd suggest we keep the same name or move it into a folder named something like `views`, though the latter makes me think there's also a `components` folder. I'd love to hear your opinion on that.

Reply:
I named it primary after the context key that the controller assigns:
```js
context.header = <Header />;
context.secondary = <PartnerPortalSidebar path={ context.path } />;
context.primary = <Licenses />; // a "/primary/" component
context.footer = <JetpackComFooter />;
next();
```
The idea being that pages that we will be adding soon (e.g. issue a license, possibly a config for OAuth tokens, web hooks etc.) will need their own primary component that will render some minimal chrome (`<Main><DocumentTitle>` etc.) and then render an actual component that has its own dedicated directory.

In summary, primary is where we stash minimal primary (page body) components that we want to hide to reduce the component noise.

Reply:
>I named it primary after the context key that the controller assigns:
Yeah, which is the "main" component of the layout. I'm just thinking that `primary` isn't as clear as other options. But `pages` or `views` would also not work, for example.
I'll approve the PR, since the changes look good, but I think we can think of a better name for that folder later.